src/Security/Authenticator/GitHubAuthenticator.php

```diff
@@ -116,9 +116,11 @@ public function start(Request $request, AuthenticationException $authException =
 
     public function onAuthenticationFailure(Request $request, AuthenticationException $exception)
     {
-        /** @var Session $session */
-        $session = $this->requestStack->getCurrentRequest()->getSession();
-        $session->getFlashBag()->add('error', 'Your GitHub user is not in the allowed organization, or the membership is private.');
+        if (null !== $request) {
+            /** @var Session $session */
+            $session = $this->requestStack->getCurrentRequest()->getSession();
```

Comment:
Oh, I did not read the code carefully the first time.
It does not make sens at all. Why using the request stack, when you already have the request? 😨
Wep, the previous code was stupid :/ 

What I was saying, BTW, because you totally miss the point: ` $this->requestStack->getCurrentRequest()` can return `null`
You new code does not make sens either. $request could NOT be null


Anyway: To fix that: use the session from $request. And if PHPStan yield, add an ignore rule in the config file